### PR TITLE
Sample UI And BroadCast Extension

### PR DESCRIPTION
**Pull request by chepury-bharathraj**

### CODE REVIEW

File: native/ios/README.md [hunk 1]
```diff
@@ -15,13 +15,6 @@ It is **strongly advised** to use the provided WebRTC framework and not
...
 with the SDK.
 
-## Required Files - Git LFS
-
-PLEASE INSTALL GIT LFS FROM THIS LOCATION - https://git-lfs.github.com/
-
-After Git LFS installation from above link, in your project directory run this command - git lfs install 
-(Note: You should above command before `pod install` command)
-
 ## Using the SDK
```
Comment (fareed069417) [inline native/ios/README.md:18]: @chepury-bharathraj  - Documentation got removed.

File: native/ios/README.md [hunk 2]
```diff
@@ -32,7 +25,7 @@ a new one following this example:
 platform :ios, '12.1'
 
 target 'MeetHourSDKTest' do
-    pod 'MeetHourSDK', '~> 3.5.2'
+    pod 'MeetHourSDK', '~> 3.5.0'
```
Comment (fareed069417) [inline native/ios/README.md:28]: @chepury-bharathraj  - Documentation got removed.

File: native/ios/README.md [hunk 3]
```diff
@@ -62,28 +55,6 @@ Last, since the SDK shows and hides the status bar based on the conference state
...
-rm -rf ~/Library/Caches/CocoaPods/*
-rm -rf ~/Library/Developer/Xcode/DerivedData/*
-pod deintegrate 
-pod setup 
-pod cache clean --all
-git lfs install // Check whether git lfs has been downloaded before install pod. See above
-pod install
-
-```
-
```
Comment (fareed069417) [inline native/ios/README.md:86]: @chepury-bharathraj  - Documentation got removed.

File: native/ios/README.md [hunk 4]
```diff
@@ -93,4 +64,4 @@ The API is documented [here](API.md).
 Please report all issues related to this SDK to the [Meet Hour]() repository.
 
 [CocoaPods]: https://cocoapods.org/pods/MeetHourSDK
-[DownloadSDK]: https://github.com/v-empower/MeetHour-MobileSDKs
+[Meet Hour]: https://github.com/v-empower/meethour-ios-sdk-releases
```
Comment (fareed069417) [inline native/ios/README.md:67]: @chepury-bharathraj  - Documentation got disturbed.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -364,47 +550,58 @@
...
+				CODE_SIGN_ENTITLEMENTS = MeetHourSDKTest/MeetHourSDKTest.entitlements;
 				CODE_SIGN_STYLE = Manual;
 				DEVELOPMENT_TEAM = 7AAZ3ZK627;
 				INFOPLIST_FILE = MeetHourSDKTest/Info.plist;
 				LD_RUNPATH_SEARCH_PATHS = (
 					"$(inherited)",
 					"@executable_path/Frameworks",
 				);
-				PRODUCT_BUNDLE_IDENTIFIER = go.meethour.io.ios.sdkTest;
+				PRODUCT_BUNDLE_IDENTIFIER = go.meethour.io.ios.sdk;
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj:586]: @chepury-bharathraj  - This bundle ID is correct.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m [hunk 6]
```diff
@@ -20,13 +20,13 @@ @implementation AppDelegate
...
-            [builder setFeatureFlag:@"ios.recording.enabled" withBoolean:YES];
-        }];
-    [MeetHour sharedInstance].defaultConferenceOptions = defaultOptions;
+//    MeetHourConferenceOptions *defaultOptions
+//        = [MeetHourConferenceOptions fromBuilder:^(MeetHourConferenceOptionsBuilder *builder) {
+//            builder.serverURL = [NSURL URLWithString:@"https://meethour.io"];
+//            builder.welcomePageEnabled = NO;
+//            [builder setFeatureFlag:@"ios.recording.enabled" withBoolean:YES];
+//        }];
+//    [MeetHour sharedInstance].defaultConferenceOptions = defaultOptions;
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m:29]: @chepury-bharathraj  - Commented these lines.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h [hunk 7]
```diff
@@ -0,0 +1,15 @@
+//
+//  CBRTextFiled.h
+//  MeetHourSDKTest
+//
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h:4]: @chepury-bharathraj  - What is CBR?

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.m [hunk 8]
```diff
@@ -0,0 +1,37 @@
+//
+//  CBRTextFiled.m
+//  MeetHourSDKTest
+//
+//  Created by Bharat Raj (VEMP) on 21/05/21.
+//  Copyright © 2021 Saúl Ibarra Corretgé. All rights reserved.
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.m:6]: @chepury-bharathraj  - What is CBR? 
Let's keep the copyright and created by to MeetHour, LLC.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/Info.plist [hunk 9]
```diff
@@ -7,7 +7,7 @@
 	<key>CFBundleExecutable</key>
 	<string>$(EXECUTABLE_NAME)</string>
 	<key>CFBundleIdentifier</key>
-	<string>$(PRODUCT_BUNDLE_IDENTIFIER)</string>
+	<string>go.meethour.io.ios.sdk</string>
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/Info.plist:10]: @chepury-bharathraj  - Let this pick from $(PRODUCT_BUNDLE_IDENTIFIER)

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/ViewController.h [hunk 10]
```diff
@@ -7,11 +7,21 @@
 //
 
 #import <UIKit/UIKit.h>
+#import "CBRTextField.h"
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/ViewController.h:10]: @chepury-bharathraj  - Let us rename this.

File: native/ios/objc/MeetHourSDKTest/MeetHour_Broadcast_Extension/SampleHandler.h [hunk 11]
```diff
@@ -0,0 +1,13 @@
+//
+//  SampleHandler.h
+//  MeetHour_Broadcast_Extension
+//
+//  Created by Bharat Raj (VEMP) on 17/05/21.
+//  Copyright © 2021 Saúl Ibarra Corretgé. All rights reserved.
+//
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHour_Broadcast_Extension/SampleHandler.h:7]: @chepury-bharathraj  - change Copyright and created by.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h (hunk 7 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h:4]: I have Created this for Padding in TextField. Just a custom class.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.m (hunk 8 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.m:6]: Okay i will change this lines.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/Info.plist (hunk 9 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/Info.plist:10]: Ok

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj (hunk 5 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj:586]: "**go.meethour.io.ios.sdk**" we have created it for **iOS**.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m (hunk 6 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m:29]: Not Required in appDelegate.

File: native/ios/swift/MeetHourSDKTest/MeetHour_Broadcast_Extension/SampleHandler.swift [hunk 12]
```diff
@@ -0,0 +1,101 @@
+//
+//  SampleHandler.swift
+//  MeetHour_Broadcast_Extension
+//
+//  Created by Bharat Raj (VEMP) on 19/05/21.
+//  Copyright © 2021 Atlassian Inc. All rights reserved.
+//
```
Comment (fareed069417) [inline native/ios/swift/MeetHourSDKTest/MeetHour_Broadcast_Extension/SampleHandler.swift:7]: @chepury-bharathraj  - change created by and copyright o Meet Hour, LLC. Follow like other pages inherited from Broadcast extension.

File: native/ios/swift/MeetHourSDKTest/Podfile [hunk 13]
```diff
@@ -1,7 +1,7 @@
 platform :ios, '12.1'
 
 target 'MeetHourSDKTest' do
-    pod 'MeetHourSDK', '~> 3.5.2'
+    pod 'MeetHourSDK', '~> 3.5.0'
```
Comment (fareed069417) [inline native/ios/swift/MeetHourSDKTest/Podfile:4]: Latest version is 3.5.2. @chepury-bharathraj

File: native/ios/swift/MeetHourSDKTest/MeetHour_Broadcast_Extension/SampleHandler.swift (hunk 12 above)
Comment (chepury-bharathraj) [reply native/ios/swift/MeetHourSDKTest/MeetHour_Broadcast_Extension/SampleHandler.swift:7]: Sure Shoeb

File: native/ios/swift/MeetHourSDKTest/Podfile.lock [hunk 14]
```diff
@@ -2,15 +2,15 @@ PODS:
   - MeetHourSDK (3.5.2)
 
 DEPENDENCIES:
-  - MeetHourSDK (~> 3.5.2)
+  - MeetHourSDK (~> 3.5.0)
```
Comment (fareed069417) [inline native/ios/swift/MeetHourSDKTest/Podfile.lock:5]: @chepury-bharathraj  - Latest version is 3.5.2

**Review by fareed069417 — CHANGES_REQUESTED** [review]
@chepury-bharathraj - Bharat make the changes marked. It includes README.md, sdk versions, Copyright and created by signatures and many other things.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj (hunk 5 above)
Comment (fareed069417) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj:586]: Ok sure. Let's stick to this. @chepury-bharathraj

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m (hunk 6 above)
Comment (fareed069417) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m:29]: Let's remove those comments.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h (hunk 7 above)
Comment (fareed069417) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h:4]: TextField.h

File: native/ios/README.md (hunk 1 above)
Comment (chepury-bharathraj) [reply native/ios/README.md:18]: Updated The doc.

File: native/ios/README.md (hunk 2 above)
Comment (chepury-bharathraj) [reply native/ios/README.md:28]: Update SDK Version to 3.5.2.

File: native/ios/README.md (hunk 3 above)
Comment (chepury-bharathraj) [reply native/ios/README.md:86]: Updated Doc.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj (hunk 5 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest.xcodeproj/project.pbxproj:586]: Done.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m (hunk 6 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m:29]: Removed.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h (hunk 7 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.h:4]: Replaced with MHTextField as it has already TextFiled(Default).

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.m (hunk 8 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/CBRTextField.m:6]: Done

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/ViewController.h (hunk 10 above)
Comment (chepury-bharathraj) [reply native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/ViewController.h:10]: MHTextField.

File: native/ios/swift/MeetHourSDKTest/Podfile (hunk 13 above)
Comment (chepury-bharathraj) [reply native/ios/swift/MeetHourSDKTest/Podfile:4]: Done.

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m [hunk 15]
```diff
@@ -2,8 +2,8 @@
 //  AppDelegate.m
 //  MeetHourSDKTest
 //
-//  Created by Saúl Ibarra Corretgé on 03/04/2019.
-//  Copyright © 2019 Saúl Ibarra Corretgé. All rights reserved.
+//  Created by to MeetHour, LLC.
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/AppDelegate.m:5]: @chepury-bharathraj - remove 'to'

File: native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/MHTextField.m [hunk 16]
```diff
@@ -0,0 +1,37 @@
+//
+//  CBRTextFiled.m
```
Comment (fareed069417) [inline native/ios/objc/MeetHourSDKTest/MeetHourSDKTest/MHTextField.m:2]: we need to change here also @chepury-bharathraj

**Review by fareed069417 — APPROVED** [review]

**Pull request merged**